### PR TITLE
[ExchangeChannels] Use exchange id

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/71
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/75

### CODE REVIEW

File: octobot_evaluators/evaluator/abstract_evaluator.py [hunk 1]
```diff
@@ -48,6 +48,9 @@ def __init__(self):
         # Evaluation related exchange name
         self.exchange_name = None
 
+        # Evaluation related exchange id
+        self.exchange_id = None
```
Comment (Herklos) [inline octobot_evaluators/evaluator/abstract_evaluator.py]: :(

Comment (Herklos) [reply octobot_evaluators/evaluator/abstract_evaluator.py]: I don't like to have exchange manager id outside of Trading modules as matrix id outside of Evaluation modules.

Comment (GuillaumeDSM) [reply octobot_evaluators/evaluator/abstract_evaluator.py]: We need a way to know to which channel an evaluator should subscribe, for trading channels we need an exchange ID. I don't have any other solution yet, do you ?
image

Comment (Herklos) [reply octobot_evaluators/evaluator/abstract_evaluator.py]: There is the same problem for matrix_id in Trading module. We need to find the same fix for this two issues.

**Pull request merged**